theme/components/icon.scss

```diff
@@ -3,22 +3,9 @@
 
 /**
  * Default width/height of an icon.
- *
- * @access public
  */
 $ck-icon-size: 20px;
 
-.ck-icon-manager__sprite {
-	display: none;
-
-	* {
-		// Allows dynamic coloring of the icons.
-		color: inherit;
-		// Needed by Webkit/Blink.
-		fill: currentColor;
-	}
-}
-
 .ck-icon {
```

Comment:
Set of icons is delivered by CKEditor, so I think that this is the Wireframe area. An icon doesn't display without width and height value, those values could be anytime overridden by the extended theme.